docs/content/tutorial.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Getting Started - A Tutorial
+order: 7
+---
+
+<a name="tutorial"></a>
+
+# Getting Started with Spectacle: A Tutorial
+
+In this guide, we'll show you a couple of different ways to get started with Spectacle and walk you through the creation and customization of a presentation deck.
+
+<a name="option-one"></a>
+
+## Option One: Using a standard React-based web app
+
+1. Spin up a new React project using [`create-react-app`](https://github.com/facebook/create-react-app):
+
+    ```sh
+    npx create-react-app spectacle-tutorial
+    ```
+
+2. Install Spectacle by running `yarn add spectacle` or `npm i spectacle`.
+
+3. In `App.js`, replace the boilerplate content with this Spectacle starter:
+
+    ```jsx
+    import React from 'react';
+    import { Deck, Slide, Heading } from 'spectacle';
+
+    function App() {
+      return (
+        <Deck>
+          <Slide>
+            <Heading>Welcome to Spectacle</Heading>
+          </Slide>
+        </Deck>
+      );
+    }
+
+    export default App;
+    ```
+
+4. And you're good to go! Using `create-react-app`'s built-in `start` script, you can start a hot-reloading server to begin building your Spectacle presentation by running `yarn run start` or `npm run start`.
+
+<a name="option-two"></a>
+
+## Option Two: Using Markdown and the Spectacle CLI
+
+1. Create a new markdown file. You can use `.md` or `.mdx` (_MDX lets you mix JSX components inside markdown).
+
+    You can use this as a starter:
+
+      ```md
+      # Welcome to Spectacle
+
+      - This is a list item
+      - This is another list item
+
+      ---
+
+      # Second Slide
+
+      Text can be **bold** or _italic_!
+
+      Notes: These are presenter notes, only visible in presenter mode to the speaker.
+      ```
+
+    **Note:** The triple dash (`---`) is used as a slide delimiter. The `Notes:` keyword is used to embed presenter notes only visible to the speaker in presenter mode.
+
+2. To view your slides, supply your markdown to the Spectacle CLI to start a local web server.
+
+    `npx spectacle -s my-slides.mdx`
```

Comment:
GLOBAL: Let's switch away from `npx spectacle` and instead do:

```js
$ npm install --global spectacle-cli
$ spectacle -s my-slides.mdx
```

While npx is technically available it is so slow (because of our deps) and I've found it doesn't cache packages so I have to wait like 30 seconds every time I do **any** `spectacle-cli`-based command.

Reply:
@ryan-roemer should I make this change in the other docs as well, so we don't recommend `npx` anywhere else?

Reply:
Good thinking -- I checked, and I think this is the only place. I already revised the spectacle-cli stuff and note that `npx` is possible, but global install is the suggested first way and used in the other examples.